proto/wayplatform/connect/mbz/v1/vehicle_specification.proto

```diff
@@ -0,0 +1,381 @@
+edition = "2023";
+
+package wayplatform.connect.mbz.v1;
+
+// VehicleSpecificationResponse is the top-level response from the Vehicle Specification API.
+message VehicleSpecificationResponse {
+  VehicleData vehicle_data = 1;
+}
+
+// CodeText represents a code with a text description.
+message CodeText {
+  string code = 1;
+  string text = 2;
+}
+
+// VehicleData represents the vehicle specification data from the Mercedes-Benz Vehicle Specification API.
+message VehicleData {
```

Comment:
there should typically only be one top-level proto message per file, and it should be named the same as the file itself - the rest should be nested messages or separate files